google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -225,6 +234,62 @@ public void downloadTo(Path path, BlobSourceOption... options) {
     }
   }
 
+  /**
+   * Builds com.google.api.services.storage.Storage using storage options.
+   *
+   * @param options storage options
+   * @return Storage
+   */
+  private com.google.api.services.storage.Storage buildStorage(StorageOptions options) {
+    HttpTransportOptions transportOptions = (HttpTransportOptions) options.getTransportOptions();
+    HttpTransport transport = transportOptions.getHttpTransportFactory().create();
+    HttpRequestInitializer initializer = transportOptions.getHttpRequestInitializer(options);
+
+    return new com.google.api.services.storage.Storage.Builder(
+            transport, JacksonFactory.getDefaultInstance(), initializer)
+        .setRootUrl(options.getHost())
+        .setApplicationName(options.getApplicationName())
+        .build();
+  }
+
+  /**
+   * Downloads this blob to the given file path using specified storage options.
+   *
+   * @param path destination
+   * @param options storage options
+   * @throws RetryHelper.RetryHelperException upon failure
+   * @throws StorageException upon failure
+   */
+  public void downloadToPathWithMediaHttpDownloader(Path path, StorageOptions options) {
+    com.google.api.services.storage.Storage storage = buildStorage(options);
+    try {
+      final com.google.api.services.storage.Storage.Objects.Get getOperation =
+          storage.objects().get(this.getBucket(), this.getName());
+      getOperation.getMediaHttpDownloader().setDirectDownloadEnabled(true);
+      final CountingOutputStream out = new CountingOutputStream(Files.newOutputStream(path));
+      runWithRetries(
+          new Callable<Void>() {
+            @Override
+            public Void call() throws Exception {
+              try {
+                getOperation.getMediaHttpDownloader().setBytesDownloaded(out.getCount());
+                getOperation.executeMediaAndDownloadTo(out);
+                return null;
+              } catch (IOException e) {
+                throw new StorageException(e);
+              }
+            };
+          },
+          options.getRetrySettings(),
+          BaseService.EXCEPTION_HANDLER,
+          options.getClock());
+    } catch (RetryHelper.RetryHelperException e) {
+      throw StorageException.translateAndThrow(e);
+    } catch (IOException e) {
+      throw new StorageException(e);
+    }
+  }
```

Comment:
I appreciate your patience.

I'd recommend passing through an additional option (USE_DIRECT_DOWNLOAD) to `downloadTo()` if using `getMediaHttpDownloader`  is considered a breaking change:

Then in the underlying RPC class handle the request in
[HttpStorageRpc.getCall()](https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java#L414) which called by [HttpStorageRpc.get()](https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java#L430). 

WDYT?

Reply:
Added USE_DIRECT_DOWNLOAD to set directDownloadEnabled for MediaHttpDownloader